string new_string
---
string status